configuration/rules-dsl.md

```diff
@@ -204,7 +204,37 @@ You can find it from PaperUI or from Karaf remote console.
 For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
-> The parser will not recognize the whole text correctly without quotes.
+
+{: #channel-based-triggers}
+### Channel-based Triggers
+
+Some add-ons provide trigger channels. 
```

Comment:
Maybe add one sentence for basic understanding? A quick draft:

> Other than classic channels, a trigger channel does not provide a continuous state but instead only informs about discrete events.

Reply:
Yes!  This is important additional information, so I vote to add it.

Consider, "Compared with other types of channels, a trigger channel provides information about discrete events, but does not provide continuous state information."